CONTRIBUTING.rst

```diff
@@ -32,5 +35,4 @@ Legal
 
 Currently we don't have any legal contributor agreement, so code
 ownership stays with the original authors. The project is licensed
-under the `Apache License Version 2
-<https://github.com/mozilla/ichnaea/blob/master/LICENSE>`_.
+under the `Apache License Version 2 <./LICENSE>`_.
```

Comment:
I think this link won't work in readthedocs.

Reply:
This file isn't included in docs, but I'll return to the full version.

Reply:
Oh, screwy. I usually include it in the docs and I must have missed this project. For example:

https://github.com/mozilla-services/socorro/blob/master/docs/contributing.rst